dali/kernels/imgproc/convolution/laplacian_cpu_test.cc

```diff
@@ -39,7 +39,6 @@ inline std::enable_if_t<!std::is_integral<Out>::value> CompareOut(
 
 void FillSobelWindow(span<float> window, int d_order) {
   int window_size = window.size();
-  assert(window_size >= 3 && window_size % 2 == 1);
```

Comment:
Why removed?

Reply:
It is called with windows_size 1 and it exploded in debug builds. Apparently it's never been run in debug build.